src/openforms/contrib/objects_api/tests/test_migrations.py

```diff
@@ -0,0 +1,29 @@
+from django.db.migrations.state import StateApps
+
+from openforms.utils.tests.test_migrations import TestMigrations
+
+
+class AddObjectsAPIGroupIdentifierTests(TestMigrations):
+    app = "objects_api"
+    migrate_from = "0001_initial"
+    migrate_to = "0002_objectsapigroupconfig_identifier"
+
+    def setUpBeforeMigration(self, apps: StateApps):
+        ObjectsAPIGroupConfig = apps.get_model("objects_api", "ObjectsAPIGroupConfig")
+        ObjectsAPIGroupConfig.objects.create(name="Group name")
+        ObjectsAPIGroupConfig.objects.create(name="Duplicate name")
+        ObjectsAPIGroupConfig.objects.create(name="Duplicate name")
+
+    def test_identifiers_generated(self):
+        ObjectsAPIGroupConfig = self.apps.get_model(
+            "objects_api", "ObjectsAPIGroupConfig"
+        )
+        groups = ObjectsAPIGroupConfig.objects.all()
```

Comment:
```suggestion
        groups = ObjectsAPIGroupConfig.objects.order_by("pk")
```
when not doing lookups by unique keys, you really need to make the ordering explicit